### ISSUE TITLE
Loudness

### ISSUE THREAD

**Issue by farski** [syndicated-media/sn-spec#36]

**Comment by farski** [comment]
I suspect there's very little debate around loudness at this point

**Comment by CharlesWiltgen** [comment]
Yep, pretty much everyone has converged on -16 LUFS for stereo content, -19 LUFS for mono (which is perceptually the same). That aligns with AES TD1004.1.15-10, recommendations from people like Thomas Lund (TC Electronic) and Auphonic, and matches loudness normalization practices as performed by Apple, Spotify, and others — basically anyone focused on mobile.

**Comment by farski** [comment]
There are a few outliers that may advocate for other values (like `-18` http://transom.org/2016/podcasting-basics-part-5-loudness-podcasts-vs-radio/). The overwhelming majority is `-16`, so if we were to pick a single value I think that's the clear winner.

**Comment by ziggythehamster** [comment]
Maybe default to -16 but include metadata in the `<item>` that allows deviation? Also, the European Broadcasting Union (EBU) has a huge trove of XML metadata that could be incorporated into RSS (and  Media RSS actually already includes one type of metadata in the form of `<media:credit>`).

**Comment by rb1111-1** [comment]
Hi all!

I wrote that Transom article linked above and was a co-author on the [AES mobile loudness](http://www.aes.org/technical/documents/AESTD1004_1_15_10.pdf) recommendation also mentioned above.

I'd encourage you to use **-18LUFS** as your normalization target! neg18, as opposed to neg16, is actually much more in line with the AES recommendation. neg16 is on the outer limit of the recommendation. neg18 also gives a little more headroom, which helps improve quality.

It's also definitely worth finding a way to work in that metadata. Our European friends will appreciate that as they try to work in their hearing protection/loudness control standards.

Do let me know how I can help!

**Comment by gholzmann** [comment]
Hallo!

I am Georg from [Auphonic](https://auphonic.com), we worked on loudness since many years and also participated in the [EBU PLOUD group](https://tech.ebu.ch/loudness) which developed the EBU R128 loudness specs.

I am a fan of the [AES mobile loudness](http://www.aes.org/technical/documents/AESTD1004_1_15_10.pdf) recommendation as well, because it includes a range (between -16LUFS and -20LUFS) and not a single value only. A maximum true peak value of -1 dBTP is also a reasonable choice.
It might be difficult to recommend a single value as many use -16LUFS, some -18, -19 or lower (iTunes Radio, Youtube, Spotify also have their own values) - however, if it's within a range, it doesn't matter that much.

Furthermore it's a nice property, that also the often used mono loudness target of -19LUFS is inside the range of the AES recommendation. The perceived loudness of a mono signal still depends on the playback device, therefore it is necessary to add -3dB on some devices, but not on all.
(Example: If you use a stereo loudness target of -16LUFS, the mono loudness target might be -19LUFS or -16LUFS.)

BTW: thanks for the syndicated.media initiative!
